auth-api/src/auth_api/services/keycloak.py

```diff
@@ -30,7 +30,7 @@ class KeycloakService:
     """For Keycloak services."""
 
     @staticmethod
-    def add_user(user: KeycloakUser, return_if_exists: bool = False):
+    def add_user(user: KeycloakUser, return_if_exists: bool = False, throw_error_if_exists: bool = False):
```

Comment:
Is there a case where we don't want to do anything if the user exists? (i.e. if both flags are False?)

If not, then these flags could be reduced to a singe flag.